src/js/features/beacon/beaconConfig.js

```diff
@@ -0,0 +1,36 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+import { printAPIError } from '../../utils/error.util.ts';
+
+export const getBeaconConfig = createAsyncThunk('beaconConfig/getBeaconConfig', async (_, { getState }) => {
+  const beaconInfoEndpoint = getState()?.config?.beaconUrl + '/info';
```

Comment:
this function isn't actually async since there are no awaits inside it

Reply:
No trouble to remove the async here. We probably don't need `createAsyncThunk` then, although this same code pattern is used throughout.

Reply:
it's still returning a promise, so probably still need createAsyncThunk - it's just that you don't await the promise, so the function doesn't need to be async